[epoch1/cuda/ee_mumu] make debug fails

[avalassi@itscrd03 bash] ~/GPU2020/upstream-madgraph4gpu/epoch1/cuda/ee_mumu/SubProcesses/P1_Sigma_sm_epem_mupmum> make debug
/usr/local/cuda-11.1/bin/nvcc  -g -O0 -DDEBUG2 -std=c++14 -I. -I../../src -I../../../../../tools -I/usr/local/cuda-11.1/include/ -DUSE_NVTX -arch=compute_70 -use_fast_math -G -c gcheck.cu -o gcheck.o
make -C ../../src debug
make[1]: Entering directory '/afs/cern.ch/user/a/avalassi/GPU2020/upstream-madgraph4gpu/epoch1/cuda/ee_mumu/src'
g++  -g -O0 -DDEBUG2 -std=c++11 -I.  -Wall -Wshadow -Wextra -fopenmp  -I/usr/local/cuda-11.1/include/ -c Parameters_sm.cc -o Parameters_sm.o
g++  -g -O0 -DDEBUG2 -std=c++11 -I.  -Wall -Wshadow -Wextra -fopenmp  -I/usr/local/cuda-11.1/include/ -c read_slha.cc -o read_slha.o
g++  -g -O0 -DDEBUG2 -std=c++11 -I.  -Wall -Wshadow -Wextra -fopenmp  -I/usr/local/cuda-11.1/include/ -c rambo.cc -o rambo.o
if [ ! -d ../lib ]; then mkdir ../lib; fi
ar cru ../lib/libmodel_sm.a Parameters_sm.o read_slha.o rambo.o  
ranlib ../lib/libmodel_sm.a
make[1]: Leaving directory '/afs/cern.ch/user/a/avalassi/GPU2020/upstream-madgraph4gpu/epoch1/cuda/ee_mumu/src'
/usr/local/cuda-11.1/bin/nvcc  -g -O0 -DDEBUG2 -std=c++14 -I. -I../../src -I../../../../../tools -I/usr/local/cuda-11.1/include/ -DUSE_NVTX -arch=compute_70 -use_fast_math -G -c gCPPProcess.cu -o gCPPProcess.o
/usr/local/cuda-11.1/bin/nvcc gcheck.o -o gcheck.exe gCPPProcess.o -arch=compute_70 -L../../lib -lmodel_sm -L/usr/local/cuda-11.1/lib64/ -lcuda -lcurand
g++  -g -O0 -DDEBUG2 -std=c++11 -I. -I../../src -I../../../../../tools -DUSE_NVTX -Wall -Wshadow -Wextra -fopenmp  -I/usr/local/cuda-11.1/include/ -c check.cc -o check.o
g++  -g -O0 -DDEBUG2 -std=c++11 -I. -I../../src -I../../../../../tools -DUSE_NVTX -Wall -Wshadow -Wextra -fopenmp  -I/usr/local/cuda-11.1/include/ -c CPPProcess.cc -o CPPProcess.o
g++ check.o -o check.exe CPPProcess.o  -g -O0 -DDEBUG2 -std=c++11 -I. -I../../src -I../../../../../tools -DUSE_NVTX -Wall -Wshadow -Wextra -fopenmp  -ldl -pthread -L../../lib -lmodel_sm -L/usr/local/cuda-11.1/lib64/ -lcuda -lcurand
g++  -g -O0 -DDEBUG2 -std=c++11 -I. -I../../src -I../../../../../tools -I../../../../../tools/googletest/googletest/include/ -DUSE_NVTX -Wall -Wshadow -Wextra -fopenmp  -I/usr/local/cuda-11.1/include/ -c runTest.cc -o runTest.o
ln -sf runTest.cc runTest_tmp.cu
/usr/local/cuda-11.1/bin/nvcc -o runTest.exe CPPProcess.o runTest.o gCPPProcess.o runTest_tmp.cu  -g -O0 -DDEBUG2 -std=c++14 -I. -I../../src -I../../../../../tools -I/usr/local/cuda-11.1/include/ -DUSE_NVTX -arch=compute_70 -use_fast_math -G -ldl -L../../lib -lmodel_sm -L../../../../../tools/googletest/build/lib// -lgtest -lgtest_main -L/usr/local/cuda-11.1/lib64/ -lcuda -lcurand -lcuda -lgomp
runTest_tmp.cu:25:10: fatal error: gtest/gtest.h: No such file or directory
 #include <gtest/gtest.h>
          ^~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:98: runTest.exe] Error 1

@hageboeck can you have a look please? This is on current master. Thanks Andrea

--- a/epoch1/cuda/ee_mumu/SubProcesses/Makefile
+++ b/epoch1/cuda/ee_mumu/SubProcesses/Makefile
@@ -27,7 +27,8 @@ ifdef CUDA_HOME
   # CUARCHFLAGS= -gencode arch=compute_$(CUARCHNUM),code=sm_$(CUARCHNUM)
   CUINC       = -I$(CUDA_HOME)/include/
   CULIBFLAGS  = -L$(CUDA_HOME)/lib64/ -lcuda -lcurand
-  CUFLAGS= $(OPTFLAGS) -std=c++14 $(INCFLAGS) $(CUINC) $(USE_NVTX) $(CUARCHFLAGS) -use_fast_math -lineinfo $(MGONGPU_CONFIG)
+  CUOPTFLAGS  = -lineinfo
+  CUFLAGS= $(OPTFLAGS) $(CUOPTFLAGS) -std=c++14 $(INCFLAGS) $(CUINC) $(USE_NVTX) $(CUARCHFLAGS) -use_fast_math $(MGONGPU_CONFIG)
   # Without -maxrregcount: baseline throughput: 6.5E8 (16384 32 12) up to 7.3E8 (65536 128 12)
   ###CUFLAGS+= --maxrregcount 160 # improves throughput: 6.9E8 (16384 32 12) up to 7.7E8 (65536 128 12)
   ###CUFLAGS+= --maxrregcount 128 # improves throughput: 7.3E8 (16384 32 12) up to 7.6E8 (65536 128 12)
@@ -61,9 +62,8 @@ endif
 
 all: ../../src $(cu_main) $(cxx_main) runTest.exe
 
-debug: OPTFLAGS = -g -O0 -DDEBUG2
-debug: CUFLAGS := $(filter-out -lineinfo,$(CUFLAGS))
-debug: CUFLAGS += -G
+debug: OPTFLAGS   = -g -O0 -DDEBUG2
+debug: CUOPTFLAGS = -G
 debug: MAKEDEBUG := debug
 debug: all
 

